docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,383 @@
+ on soon
+- **Label-specific views**: consisting of issues for specific labels, like
+ documentation or a large refactor
+
+Right now, the team at Voltron Data sets the roadmap and priorities. Over time
+as more contributors and organizations join the project, we expect this process
+to diversify and become more community-driven.
+
+### Overall themes
+
+Our top five themes for 2024 include:
+
+1. **Ibis backends**: Ibis is a Python frontend for many backends. To continue
+scaling to more backends, we need to complete a major rework of library
+internals and stabilize the API for backend authors. Related work in this area
+will make it easier than ever to create new Ibis backends and maintain them.
+This work will also include improving backend interfaces for DDL operations like
+table creation, insertion, and upsertion.
+
+2. **Ibis for ML**: Increasingly, data projects are ML projects. Ibis can
+uniquely help with feature engineering and other ML tasks connecting your data
+where it lives to ML models. We will continue to improve Ibis for ML use cases.
+
+3. **Ibis for streaming data**: Ibis has only been for batch data until very
+recently. With the addition of the first streaming backends, we will continue to
+improve Ibis for streaming data use cases and bridge the gap between batch and
+streaming data.
+
+4. **Ibis for geospatial**: Ibis has a rich set of geospatial expressions, but
+most backends do not implement them. We will continue to improve Ibis for
+geospatial use cases and bridge the gap between geospatial data and other data
+types.
+
+5. **Ibis community**: Ibis is an open-source project and we want to make it as
+easy as possible for new contributors to get involved. We will continue to
+improve the Ibis community and make it easier than ever to contribute to Ibis.
+
+We believe these themes will help Ibis as a standard Python interface for many
+backends and real-world data use cases.
+
+### The big refactor
+
+The biggest item in Q1 2024 and primary focus of the core Ibis team right now is
+the big refactor -- dubbed "the epic split" -- continuing the great work
+completed by Krisztián in [his PR splitting the relational
+operations](https://github.com/ibis-project/ibis/pull/7752). You can read more
+details in that PR, but the gist is that a new intermediary representation for
+Ibis expressions is being has been created that drastically simplifies the
+codebase.
+
+With that refactor in place, each backend Ibis supports needs to be moved to the
+new relational model. As a consequence, we are also swapping out SQLAlchemy for
+[SQLGlot](https://github.com/tobymao/sqlglot). We are losing out on some of the
+things SQLAlchemy did for us automatically, but overall this gives us a lot more
+control over the SQL that is generated, reduces dependency overhead, and
+simplifies the codebase further.
+
+::: {.callout-note}
+By the time this is published, the refactor will likely be complete and on the
```

Comment:
When are we planning on publishing this? Not sure if the "By the time this is published, the refactor..." statement will be true.

Reply:
yeah let me perhaps adjust